bundles/NotificationBundle/Views/Popup/index.html.php

```diff
@@ -211,4 +211,4 @@ function showError(error) {
         document.getElementById(error).style.display = "block";
     }
 </script>
-</body></html>
\ No newline at end of file
+</body></html>
```

Comment:
This change seems unnecessary

Reply:
The new line character added at the end of the file.